sympy/matrices/expressions/matexpr.py

```diff
@@ -642,7 +639,6 @@ class MatrixSymbol(MatrixExpr):
     I + 2*A*B
     """
     is_commutative = False
-    is_symbol = True
```

Comment:
why?

Reply:
I just clicked revert on the PR. If there are other unrelated changes from that PR that should be kept intact we should keep them.